src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/builder/BomEditMessageBuilder.java

```diff
@@ -141,12 +142,14 @@ public BomEditMessageBuilder(PolicyCommonBuilder policyCommonBuilder) {
                 ComponentItem.Builder builder = new ComponentItem.Builder()
                                                     .applyCategory(MessageBuilderConstants.CATEGORY_TYPE_VULNERABILITY)
                                                     .applyOperation(ItemOperation.UPDATE)
-                                                    // FIXME get the vulnerability id(s) and create a ComponentItem from each of them
+                                                    // TODO get the vulnerability id(s) and create a ComponentItem from each of them
```

Comment:
Switched to TODO  because FIXME implies we shouldn't release until this is resolved IMO.